apps/native/components/EmployeeList.tsx

```diff
@@ -0,0 +1,284 @@
+import React, {useState} from 'react'
+import {doc, getDoc, setDoc, addDoc, collection} from "firebase/firestore";
+import {auth, db} from "@native/firebase";
+import {createUserWithEmailAndPassword} from "@firebase/auth";
+import {
+    KeyboardAvoidingView,
+    Modal,
+    Platform,
+    SafeAreaView,
+    StyleSheet, Text, TextInput,
+    TouchableOpacity,
+    TouchableWithoutFeedback,
+    View,
+} from 'react-native';
+import FontAwesome5 from "react-native-vector-icons/FontAwesome5";
+import DropDownPicker from "react-native-dropdown-picker";
+
+export const EmployeeList = ({propertyId}) => {
+
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const role = "Employee";
+    const [job, setJob] = useState("Janitor");
+    const [passwordVisible, setPasswordVisible] = useState(false);
+    const [connectionStatus, setConnectionStatus] = useState("");
+    const [errorMessage, setError] = useState("");
+    const [visible, setVisible] = useState(false);
+    const [open, setOpen] = useState(false);
+    const show = () => setVisible(true);
+    const hide = () => setVisible(false);
+
+    const handleRegistration = async () => {
+        async function returnRole(uid: string) {
+            const docRef = doc(db, "users", uid);
+            const docSnap = await getDoc(docRef);
+
+            if (docSnap.exists()) {
+                console.log("Document data:", docSnap.data());
+                return docSnap.data().role;
+            } else {
+                // docSnap.data() will be undefined in this case
+                return "notFound";
+            }
+        }
+        try {
+            const userCredential = await createUserWithEmailAndPassword(auth, email, password);
+            const user = userCredential.user;
+
+            await setDoc(doc(db, "users", user.uid), {
+                email: email,
+                role: role,
+                job: job,
+            });
+
+            await addDoc(collection(db, `properties/${propertyId}/Employees`), {
+                user: doc(db, "users", user.uid)
```

Comment:
Just to be sure, shouldn't the employees be assigned to condo management company instead of properties? I think it would be easier to keep it like that tho (assign to the properties)